go.mod

```diff
@@ -64,3 +64,5 @@ require (
 	gopkg.in/go-playground/validator.v9 v9.20.2
 	honnef.co/go/tools v0.0.1-2020.1.3 // indirect
 )
+
+replace github.com/mattn/go-ieproxy v0.0.0-20190610004146-91bb50d98149 => github.com/mattn/go-ieproxy v0.0.1
```

Comment:
I don't think we want a replace here, since it will make Athens not go get-able.

Reply:
Will it really? I could have sworn I read somewhere in the module code that replace was respected by `go get`.. maybe not though..

Reply:
https://github.com/golang/tools/blob/master/gopls/go.mod#L13 `go get golang.org/x/tools/gopls` still works, for example.

Reply:
It will still be `go get`-able (assuming the build works), but replaces are currently ignored.  IIRC there is a Go proposal or issue which would cause `go get` to respect the replaces when it is run outside of any module and is being used to install an executable, but only for exact version replaces (like here) and not local filepath replaces (beginning with . or ..).

Reply:
Yep, for those reasons and since the jury is out on how replaces will work (i believe there's another proposal to error out on replace clauses), we should shy away from having replace clauses in our go.mod file. 

@qbit I'm happy to keep this dependency as an explicit requirement like before. 

I think the only thing we were wondering about is details: how does this dependency work? And how come it works by just updating go.mod file without any code in our Athens repository that explicitly calls it?

Reply:
@marwan-at-work the reason it works now is the `+build` like in `proxyMiddleman_unix.go` was (a1ba01f) `darwin unix linux`.  Later in https://github.com/mattn/go-ieproxy/commit/91bb50d981495aef1c208d31be3d77d904384f20 the definition of `proxyMiddleman()` moved to `proxy_middleman_unix.go` and the `+build` was changed to `!windows` - thus including OpenBSD.